src/components/issue.diff

```diff
@@ -0,0 +1,848 @@
+diff --git a/src/components/issue.jsx b/src/components/issue.jsx
```

Comment:
Unrelated file

Reply:
Thx. A stupid mistake which should definitely be avoided. 😅